jsonHandler.py

```diff
@@ -0,0 +1,149 @@
+import json
+import datetime
+from events import *
+
+class JsonHandler(object):
+
+    def __init__(self):
+        self._filename = "json_data.json"
+
+
+    def add_events(self,event):
+        data = None
+        with open(self._filename,'r') as data_file:
+            data = json.load(data_file)
+        d={str(event.get_event_id()):{"name":str(event.get_name()),"info":str(event.get_info()),"date":str(event.get_date()),"venue":str(event.get_venue()),"city":str(event.get_city())}}
```

Comment:
expose a method toJSON() in Event, which returns the event as dict, so that it is reusable